src/applications/vaos/sass/vaos.scss

```diff
@@ -29,7 +27,8 @@ h1:focus {
 
 }
 .vaos__loading-overlay {
-  background: $vaos-loading-overlay;
+  background-color: var(--vads-color-white);
+  opacity: 0.9;
```

Comment:
I don't think this is needed any more. Only used in 'LoadingOverlay.jsx'.

Reply:
I'll leave it there just in case we ever have to use the style